core/src/initializers/codeConfig.ts

```diff
@@ -16,7 +16,7 @@ export class CodeConfig extends CLSInitializer {
     super();
     this.name = "codeConfig";
     this.loadPriority = 10000;
-    this.startPriority = 1;
+    this.startPriority = 100;
```

Comment:
Can you please remind me why we made this change?

Reply:
There was a sequelize instance error because calculated-properties plugin needs access to one of the models